internal/echo.go

```diff
@@ -0,0 +1,14 @@
+package internal
+
+import (
+	"strings"
+)
+
+func Echo(n bool, input string) string {
+	
+	if n {
+		return strings.Replace(input, "\r\n", "", -1)
+	}
+	
+	return input
```

Comment:
again this is not the functionality of the echo, the internal functions should do more than reformatting